### ISSUE TITLE
error

### ISSUE THREAD

**Issue by morandave** [ygjwd12345/TransDepth#23]
I tried to run python ’bts_test.py arguments_test_nyu.txt‘, but returned 'No module named 'bts_nyu_v2_pytorch_att''

**Comment by ygjwd12345** [comment]
Sorry for reply late. I think you need to carefully check  model name in training set and test set.

**Comment by morandave** [comment]
This is the path of my training set: "TransDepth-main/pytorch/dataset/nyu_depth_v2/official_splits/test/" ,I tried "nyu_depth_v2",but failed.Return "No module named 'nyu_depth_v2'"

**Comment by ygjwd12345** [comment]
![image](https://github.com/ygjwd12345/TransDepth/assets/32895754/275c493e-2278-4365-ae0f-7e69541daf9c)
check model name in these kind of setting files.
